### PR TITLE
add SE subzone exchanges

### PR DESCRIPTION
**Pull request by maxbellec**

### CODE REVIEW

**Comment by maxbellec** [comment]
while at it, should I also add exchanges for `NO` subzones?

File: parsers/ENTSOE.py [hunk 1]
```diff
@@ -69,6 +69,7 @@
     'CH': '10YCH-SWISSGRIDZ',
     'CZ': '10YCZ-CEPS-----N',
     'DE': '10Y1001A1001A83F',
+    'DE-AT-LU': '10Y1001A1001A63L',
```
Comment (corradio) [inline parsers/ENTSOE.py]: Cool. Can you update `ENTSOE_PRICE_DOMAIN_OVERRIDE` now that you added this dict entry?

File: parsers/ENTSOE.py [hunk 2]
```diff
@@ -127,6 +128,8 @@
 ENTSOE_EXCHANGE_DOMAIN_OVERRIDE = {
     'DE->DK-DK1': ['10Y1001A1001A63L', ENTSOE_DOMAIN_MAPPINGS['DK-DK1']],
     'DE->DK-DK2': ['10Y1001A1001A63L', ENTSOE_DOMAIN_MAPPINGS['DK-DK2']],
+    'DE->SE-SE4': [ENTSOE_DOMAIN_MAPPINGS['SE-SE4'],
```
Comment (corradio) [inline parsers/ENTSOE.py]: Any need for adding `PL->SE-SE4`, `FI->SE-SE{1,3}`and `LT->SE-SE4`?
I'm not sure the exchanges between bidding zones and countries will work.

Comment (maxbellec) [reply parsers/ENTSOE.py]: no need to add, it works

**Pull request merged**